pallets/hyperdrive-outgoing/src/lib.rs

```diff
@@ -288,11 +339,13 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
     fn node_temp_offchain_key(
         pos: NodeIndex,
         parent_hash: <T as frame_system::Config>::Hash,
+        unique: HashOf<T, I>,
```

Comment:
I do not see the need for the `unique` argument. The combination of `pos` and `parent_hash` should already be unique, right?